### PR TITLE
Test 18 may 

### PR DESCRIPTION
**Pull request by sabuchan07**
Cherry-pick into:

* [ x] Foreman 3.3
* [x ] Foreman 3.2
* [ ] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Hi @sabuchan07 
Please take care of the following items before we can properly review your PR:

* Choose a proper title of your PR
* Ensure to not check in any unrelated files, like `.DS_Store` from macOS
* Ensure to comply with our conventions, for example "one sentence per line".
* Ensure to use attributes instead of hardcoded versions.
* Rebase (+squash) your changes and use a proper commit message. For example, community users like myself don't have access to SAT tickets. Referencing them in the "description" part of a git commit message is fine, but the message itself must reflect what has been changed and why. See https://cbea.ms/git-commit/#seven-rules
* Think about the word "should". > See IBM Style guide

File: guides/common/assembly_installing-satellite-server-disconnected.adoc [hunk 1]
```diff
@@ -15,6 +15,9 @@ endif::[]
 
 NOTE: You cannot register {ProjectServer} to itself.
 
+NOTE: If you have a disconnected environment and some of your {ProjectServer}s are on versions older than 6.10 while others are on 6.10, content import and content export do not work.
```
Comment (melcorr) [inline guides/common/assembly_installing-satellite-server-disconnected.adoc:18]: Use attributes.

File: guides/common/assembly_installing-satellite-server-disconnected.adoc [hunk 2]
```diff
@@ -15,6 +15,9 @@ endif::[]
...
+All {ProjectServer}s should be on the same version.
```
Comment (melcorr) [inline guides/common/assembly_installing-satellite-server-disconnected.adoc:19]: Follow IBM style conventions and do not use should.

File: guides/common/modules/con_importing-content.adoc [hunk 3]
```diff
@@ -9,6 +9,9 @@ ifdef::orcharhino[]
 If you want to import RPM or DEB packages, Files, or Puppet Modules to {Project}, it is done with largely the same procedures in this chapter.
 endif::[]
 
+NOTE: If you have a disconnected environment and some of your {ProjectServer}s are on versions older than 6.10 while others are on 6.10, content import and content export do not work.
```
Comment (melcorr) [inline guides/common/modules/con_importing-content.adoc:12]: Please do not litter the docs with notes. If you read our docs, we are very careful about the use of notes. 
Please respect that this is a single sourced content that is used for multiple projects and products. You adding version numbers for a Red Hat product is not helpful to our upstream or other contributors.

**Review by melcorr — COMMENTED** [review]
Please try to help us help you @sabuchan07 
A title like test is very confusing and wastes the time of our contributors.

**Pull request closed without merging**